packages/firebase_data_connect/firebase_data_connect/example/dataconnect/out.txt

```diff
@@ -0,0 +1,161 @@
+I0701 15:44:57.834681      67 codegen.go:55] [connector "movies" Javascript SDKs] Start Generating Sources into "../../web"...
```

Comment:
Maybe we can remove and gitignore this file?

Reply:
Removed.